pkg/apis/serving/v1alpha1/trained_model_status.go

```diff
@@ -17,13 +17,17 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 )
 
 // TrainedModelStatus defines the observed state of TrainedModel
 type TrainedModelStatus struct {
 	// Conditions for trained model
 	duckv1.Status `json:",inline"`
+	// URL holds the url that will distribute traffic over the provided traffic targets.
+	// http[s]://{route-name}.{route-namespace}.{cluster-level-suffix}/v1/models/<trainedmodel>:predict
```

Comment:
For v2 it should be http[s]://{route-name}.{route-namespace}.{cluster-level-suffix}/v2/models/<trainedmodel>:infer